docs/src/domains/steps/dns.md

```diff
@@ -1,170 +1,139 @@
-  Routers are taken offline, one at a time, to apply the changes.
-
-The edge hostname's destination IP addresses are updated automatically should they change.
-
-If a router is being upgraded and its IP changed, two possibilities arise:
-
-* Your apex domain points to the edge hostname using `CNAME`/`ANAME` or `ALIAS` records. The IP addresses for the routers are updated automatically. You don't need to do anything. Your website remains online.
-* Your apex domain points to your project's region using `A` records.
-  The IP addresses for the routers aren't updated automatically.
-  Your website appears temporarily offline until you manually update your `A` records or the router is back from maintenance.
-
-The edge hostname can be [retrieved through the CLI or the Console](./_index.md#2-get-the-target-for-your-project).
-
-## Why `CNAME` records are problematic
-
-The DNS specification was originally published in 1987, long before name-based HTTP hosting became prevalent.
-In the multiple RFCs that were written regarding `CNAME` records, the description of their behavior is rather vague.
-
-It's generally understood that a `CNAME` record for an apex domain like `example.com`:
-
-* Can only point to an IP address like `192.0.2.1` (an `A` record).
-* Can't be used as an alias for another hostname like `www.example.com` (a `CNAME` record).
-
-The `CNAME` record limitation is especially problematic if you want to use an apex domain with any container-based managed hosting service like Platform.sh.
-
-Many registrars allow `CNAME` records for apex domains.
-If yours doesn't, several solutions exist [to bypass that limitation](#handling-apex-domains).
-
-## Handling Apex domains
-
-Some DNS providers (usually your registrar) don't allow `CNAME` records for [apex domains](../../other/glossary.md#apex-domain).
-This is one of the [limitations to `CNAME` records](#why-cname-records-are-problematic).
-
-Check your registrar's documentation to make sure that `CNAME` records on apex domains are supported.
-If your registrar supports them, follow the [guide to using such records](../steps/_index.md).
-If your registrar doesn't support them, there are a number of ways to handle the limitation.
-
+To map incoming requests to the appropriate container,
+Platform.sh runs routers in [each region](../../development/regions.md).
+A router's IP address can change in two cases:
+- During an upgrade or maintenance operation, routers can be taken offline while changes are applied.
+- During a region upscale or downscale, routers can be added or removed.
+
+To prevent downtime in such situations,
+create `CNAME` records through your DNS provider.
+ 
+`CNAME` records map a domain to another.
+Set up your `CNAME` records to point at your project's [target edge hostname](../../domains/steps/_index.md#2-get-the-target-for-your-project).
+Each Platform.sh edge hostname comes with a DNS record
+that resolves the IP addresses of the routers in your project's region.
```

Comment:
Don't really like this sentence, but couldn't find a way to make it clearer so definitely open to suggestions.